src/engines/Cesium/hooks.ts

```diff
@@ -665,8 +665,8 @@ export default ({
       if (!onCreditsUpdateRef.current) return;
       const viewer = cesium.current?.cesiumElement;
       if (!viewer || viewer.isDestroyed()) return;
-      const credits: Credit[] = getCredits(viewer) ?? [];
-      onCreditsUpdateRef.current(credits);
+      const credits = getCredits(viewer);
+      onCreditsUpdateRef.current(credits as Credits);
```

Comment:
Better avoid using `as` when there's a real type issue, i guess it could be `undefined`, if you just want to fix this please update the type of `onCreditsUpdate` making it accept undefined as well.